build.sh

```diff
@@ -4,6 +4,9 @@ set -e
 
 ORG_PATH="github.com/appc"
 REPO_PATH="${ORG_PATH}/docker2aci"
+# TODO: use "git describe --dirty" when we actually have tags pushed
+VERSION=$(git rev-parse --short HEAD)
+GLDFLAGS="-X github.com/appc/docker2aci/lib.Version=${VERSION}"
```

Comment:
Yeah, I don't mind if docker2aci drops support for go1.4.

About version - not sure if it is best of both worlds or worst of both worlds. But that will work too, yes.